tests/console/salt.pm

```diff
@@ -15,20 +15,24 @@
 use base "consoletest";
 use strict;
 use testapi;
+use utils 'pkcon_quit';
 
 sub run() {
+    my $output = "salt '*' test.ping | grep -woh True > /dev/$serialdev";
     select_console 'root-console';
+    pkcon_quit;
     my $cmd = <<'EOF';
 zypper -n in salt-master salt-minion
 systemctl start salt-master
 systemctl status salt-master
 sed -i -e "s/#master: salt/master: localhost/" /etc/salt/minion
 systemctl start salt-minion
-yes | salt-key --accept-all ; [ "$?" = "141" ] || [ "$?" = "0" ]
-salt '*' test.ping
-systemctl stop salt-master salt-minion
+systemctl status salt-minion
+salt-key --accept-all -y
 EOF
     assert_script_run($_) foreach (split /\n/, $cmd);
+    validate_script_output $output, sub { m/True/ };
```

Comment:
Isn't that the same as just calling `salt '*' test.ping | grep -woh True` ? The grep will exit successfully if there *is* true so why so complicated?